addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -892,6 +892,7 @@ export class OdooEditor extends EventTarget {
                 for (const node of record.removedNodes) {
                     this.idSet(node, this._checkStepUnbreakable);
                     mutatedNodes.delete(node.oid);
+                    node.ouid = undefined;
```

Comment:
Are you sure this one is necessary? The new unit test you wrote doesn't fail if I remove this line.
If it's necessary it would be great to have a unit test to cover it.

Reply:
You are right , this is not necessary, I will remove it